src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -669,6 +669,38 @@ describe('Testing Edit concept actions ', () => {
     });
   });
 
+  it('should unretire a Mapping', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({ status: 200, response: existingConcept });
+    });
+    const expectedActions = [
+      { type: IS_FETCHING, payload: true },
+      { type: UPDATE_CONCEPT, payload: existingConcept },
+    ];
+    const store = mockStore(mockConceptStore);
+    const conceptUrl = '/orgs/EthiopiaNHDD/sources/HMIS-Indicators/concepts/C1.1.1.1/';
+    return store.dispatch(unretireMapping(conceptUrl)).then(() => {
+      expect(store.getActions()).toEqual(expectedActions);
+    });
+  });
+
+  it('should handle exceptions for unretire a Mapping', () => {
```

Comment:
The same as above.

Reply:
It was affecting my coverage but I have now created a separate [PR](https://github.com/openmrs/openmrs-ocl-client/pull/382) for it.